snap/snapcraft.yaml

```diff
@@ -813,12 +813,22 @@ parts:
   device-virtual-go:
     source: https://github.com/edgexfoundry/device-virtual-go.git
     source-depth: 1
-    # TODO: ideally this should be changed to v2.0.0
-    source-tag: v2.0.0
+    source-branch: v2.1.0
```

Comment:
Please add a comment to explain why setting branch to tag was necessary. E.g.:
```
Set source-branch to tag so that depth of 1 is relative to that tag.
This is equivalent to "git clone --depth 1 --branch <tag> <url>"
```

Reply:
Thanks a lot for your review. Updated.